build.sh

```diff
@@ -20,6 +20,7 @@ for build in ${BUILDS[@]}; do
         --build-arg node_protoc_gen_grpc_web_version=${NODE_PROTOC_GEN_GRPC_WEB_VERSION} \
         --build-arg go_envoyproxy_pgv_version=${GO_ENVOYPROXY_PGV_VERSION} \
         --build-arg go_mwitkow_gpv_version=${GO_MWITKOW_GPV_VERSION} \
+        --build-arg ts_proto_version=${TS_PROTO_VERSION} \
```

Comment:
Just to keep our linter happy for this PR (this is something we will address for all other similar code separately)
```suggestion
        --build-arg ts_proto_version="${TS_PROTO_VERSION}" \
```